package.json

```diff
@@ -30,9 +30,12 @@
     "@react-native-community/async-storage": "^2.0.0-rc.1",
     "@react-native-community/async-storage-backend-legacy": "^2.0.0-rc.1",
     "@react-native-community/masked-view": "^0.1.10",
-    "@react-native-firebase/analytics": "^7.6.7",
-    "@react-native-firebase/app": "^8.4.5",
-    "@react-native-firebase/remote-config": "^9.0.10",
+    "@react-native-firebase/analytics": "^10.5.1",
```

Comment:
These require cocoapods 1.10.0, no? Which is dependent on #925 ?

Reply:
I do not get an error regarding 1.10.0 when using these, so would be cool if you could pull branch local and test :o

Reply:
Seems as `@react-native-firebase/app@10.8.0` depends on `Firebase/CoreOnly ~> 7.6.0` which in turn depends on pods 1.10.0. But `10.5.0` as used here is good. Though there are some inconsistencies when merging with the main branch now.